packages/posthog/android/src/main/java/io/capawesome/capacitorjs/plugins/posthog/classes/results/GetFeatureFlagPayloadResult.java

```diff
@@ -0,0 +1,30 @@
+package io.capawesome.capacitorjs.plugins.posthog.classes.results;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import com.getcapacitor.JSObject;
+import io.capawesome.capacitorjs.plugins.posthog.interfaces.Result;
+import org.json.JSONObject;
+
+public class GetFeatureFlagPayloadResult implements Result {
+
+    @Nullable
+    private final Object value;
+
+    public GetFeatureFlagPayloadResult(@Nullable Object value) {
+        this.value = value;
+    }
+
+    @NonNull
+    public JSObject toJSObject() {
+        JSObject result = new JSObject();
+        if (value instanceof Boolean) {
+            result.put("value", (Boolean) this.value);
+        } else if (value instanceof String) {
+            result.put("value", (String) this.value);
+        } else {
+            result.put("value", JSONObject.NULL);
+        }
```

Comment:
```suggestion
        if (value == null) {
            result.put("value", JSONObject.NULL);
        } else {
            result.put("value", this.value);
        }
```

Shouldn't that also work? After all, there are more types than just string and boolean, right?

We are doing something similar here: https://github.com/capawesome-team/capacitor-firebase/blob/61a15cb4d12f5132569d2e8a681c0fd31838e2de/packages/firestore/android/src/main/java/io/capawesome/capacitorjs/plugins/firebase/firestore/FirebaseFirestoreHelper.java#L51